app/assets/javascripts/sign_up_async_request.js.erb

```diff
@@ -78,6 +78,21 @@ function handleError(){
 	$("#check_email_span").text("We've encountered an error. Please try again later or email help@mylibrarynyc.org for assistance.");
 }
 
+function validEmailMessage(){
```

Comment:
`showValidEmailMessage` might be a bit more accurate for what it does.

Reply:
Maybe pass in a boolean value about whether the validation passed or failed, and then you can fully control the disabling of the form's submit button here.